src/domain/shared/components/ActivityLog/views/ActivityLogDiscussionCommentCreatedView.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from 'react';
```

Comment:
Defining it as you suggest would require the `ActivityLogComponent` to be aware of every action string of every view (including future once) the `ActivityViewChooser` will switch between.
This is simpler than what you suggest because the interface is simplified and there isn't custom logic to address each version of the action string.